.gitignore

```diff
@@ -4,4 +4,4 @@
 # or operating system, you probably want to add a global ignore instead:
 #   git config --global core.excludesfile '~/.gitignore_global'
 
-.env
```

Comment:
We may wanna keep `.env`, just to keep it from getting accidentally checked in by anyone who's cloned this.